spec/test_app/test_suite.sh

```diff
@@ -2,5 +2,4 @@
 
 cd $(dirname $0)
 
-find scripts -type f -executable -exec '{}' \;
-
+find scripts -type f -perm -111 -exec bash '{}' \;
```

Comment:
Here's an excerpt of the man of _GNU find_:

```
       -perm -mode
              All of the permission bits mode are set for the file.  Symbolic modes are accepted in this form, and this is usually the way in which would want to use them.  You must specify `u', `g' or `o'
              if you use a symbolic mode.   See the EXAMPLES section for some illustrative examples.

              
       -perm /mode
              Any of the permission bits mode are set for the file.  Symbolic modes are accepted in this form.  You must specify `u', `g' or `o' if you use a symbolic mode.  See the  EXAMPLES  section  for
              some illustrative examples.  If no permission bits in mode are set, this test matches any file (the idea here is to be consistent with the behaviour of -perm -000).


       -perm +mode
              This is no longer supported (and has been deprecated since 2005).  Use -perm /mode instead.
```

Note that the `-executable` flag in _GNU find_ tests that the file can be executed by the current user. Since we can't test this specifically with `-perm`, I think it'd be more bulletproof to use `-perm /111`. But I wonder if that is supported on OS X?